clients/drcachesim/scheduler/scheduler.h

```diff
@@ -805,6 +805,15 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * (#block_time_max_us) scaled by #block_time_multiplier.
          */
         bool honor_infinite_timeouts = false;
+        /**
+         * For #MAP_TO_ANY_OUTPUT, when an input reaches EOF, if the number of non-EOF
+         * inputs left as a fraction of the original inputs is equal to or less than
+         * this value then the scheduler exits (sets all outputs to EOF) rather than
+         * finishing off the final inputs.  This helps avoid long sequences of idles
+         * during staggered endings with fewer inputs left than cores and only a small
+         * fraction of the total instructions left in those inputs.
+         */
+        double exit_if_fraction_left = 0.1;
```

Comment:
This was meant to be 0.0 as the PR description says; will change along w/ review requests.

Reply:
Renamed to "exit_if_fraction_inputs_left".  And set to 0.05 to match the CLI and not diverge.